test/configSpec.js

```diff
@@ -189,4 +189,23 @@ describe('loadConfig', () => {
       expect(loadConfig).to.throw(/Cannot support precision > 15/)
     })
   })
+
+  describe('config.recommendedConnectors', () => {
+    it('defaults to undefined', () => {
+      const _config = loadConfig()
+      expect(_config.recommendedConnectors).to.equal(undefined)
+    })
+
+    it('is undefined when recommendedConnectors is "*"', () => {
```

Comment:
Any accounts listed in `RECOMMENDED_CONNECTORS` that aren't actually connector's will be excluded from the returned list. If they are later changed to connectors, they will get added in automatically.